### ISSUE TITLE
Contact Merge can give error if Duplication Management is enabled

### ISSUE THREAD

**Issue by judisohn** [SalesforceFoundation/Cumulus#1669]
Hub thread: https://powerofus.force.com/0D58000002DMsnR

Example:

Update failed. First exception on row 0 with id 00324000008sgjRAAQ; first error: DUPLICATES_DETECTED, You're creating a duplicate record. We recommend you use an existing record instead.: []
Failed to merge the contacts. Error: System.DmlException: Update failed. First exception on row 0 with id 00324000008sgjRAAQ; first error: DUPLICATES_DETECTED, You're creating a duplicate record. We recommend you use an existing record instead.:

Additional info from Dave:

basically, our code updates the winning contact's fields, and then call's Salesforce's merge api to move the child records, and then our TDTM code does a bunch of fixup work for things Salesforce Merge won't take care of.

**Comment by mrbelvedere** [comment]
Included in beta release version 3.52 (Beta 4) and higher

**Comment by mrbelvedere** [comment]
Included in production release version 3.52
